src/Classes/ImportTab.lua

```diff
@@ -81,11 +81,11 @@ local ImportTabClass = newClass("ImportTab", "ControlHost", "Control", function(
 	self.controls.sessionHeader.label = function()
 		return [[
 ^7The list of characters on ']]..self.controls.accountName.buf..[[' couldn't be retrieved. This may be because:
-1. The account name is wrong, or
```

Comment:
technically this is more correct, as in some cases it is a wrong account name and not always just entering a character name instead of an account name, so this line should probs be left unchanged

Reply:
it covers more cases, but I feel like 95% of the time I watch people do this, it's because they punched in their character name. I think everyone already understands that if you type in the wrong name, it doesn't work. I think we should aim to be helpful rather than as correct as possible. happy to change it back if you still disagree